README.md

```diff
@@ -1,5 +1,6 @@
 # MIRT (Michigan Image Reconstruction Toolbox) in Julia
 
+[![][action-img]][action-url]
 [![Build Status][travis-img]][travis-url]
```

Comment:
Given that Travis is not used anymore for unit test, this badge can be removed 😄

Reply:
done